kb-api-swagger/src/main/swagger/kb-api.yml

```diff
@@ -85,6 +85,188 @@ paths:
+              value: ""
+            oneSortField:
+              summary: Example of a single sort field
+              value: "score desc"
+            multipleSortFields:
+              summary: Example of a multiple sort fields
+              value: "score desc, width_pixels asc"
+        - name: start
+          in: query
+          description: Number of leading documents to skip.
+          required: false
+          schema:
+            type: integer
+            default: 0
+          example: 0
+        - name: rows
+          in: query
+          description: Max results per page. This sets the "page size" for paginated search results.
+          required: false
+          schema:
+            type: integer
+            default: 10
+        - name: fl
+          in: query
+          description: Field List. The list of fields to return in the query results.
+          required: false
+          schema:
+            type: string
+            default: ""
+          allowReserved: true
+          examples:
+            emptyField:
+              summary: Empty field
+              value: ""
+            oneField:
+              summary: Example of a single field
+              value: "author"
+            multipleFields:
+              summary: Example of a multiple fields
+              value: "author, title"
+        - name: df
+          in: query
+          description: Default field. Used to configure the q and fq parameters. If not specified, the default field is text
+          required: false
+          schema:
+            type: string
+            default: ""
+        - name: wt
+          in: query
+          description: Response writer.
+          required: false
+          schema:
+            type: string
+            default: "json"
+            enum: [json, xml]
+        - name: facet
+          in: query
+          description: Enable faceting. If set to true, this parameter enables facet counts in the query response. If set to false, this parameter disables faceting. None of the other parameters listed below will have any effect unless this parameter is set to true. The default value is blank (false).
+          required: false
+          schema:
+            type: boolean
+            default: false
+        - name: facet_query
+          in: query
+          description: This parameter allows you to specify an arbitrary query in the Lucene default syntax to generate a facet count.
+          required: false
+          schema:
+            type: string
+            default: ""
+          allowReserved: true
+          examples:
+            emptyFacetQuery:
+              summary: Example of a empty facet query
+              value: ""
+            oneFacetQuery:
+              summary: Example of a single facet query
+              value: "type:text"
+            multipleFacetQuery:
+              summary: Example of a multiple facet query
+              value: ""
+        - name: facet_field
```

Comment:
The Solr-parameters use dots, not underscores as separaters for keys. Please change to `facet.field` instead of `facet_field`.

Reply:
Var ikke sikkert om Swagger kan håndtere punktum, men det kan det godt, så er det rettet